prql-compiler/src/error.rs

```diff
@@ -24,8 +24,8 @@ pub struct Error {
 
 /// Location within the source file.
 /// Tuples contain:
-/// - line number (1-based),
-/// - column number within that line (1-based),
+/// - line number (0-based),
```

Comment:
Should we change this to 1-based? It seems quite non-standard, no?

(as ever — not saying you have to do it)